base/shell/explorer/lang/nl-BE.rc

```diff
@@ -0,0 +1,200 @@
+LANGUAGE LANG_ENGLISH, SUBLANG_ENGLISH_US
+
+IDM_TRAYWND MENU
+BEGIN
+    POPUP ""
+    BEGIN
+        MENUITEM SEPARATOR
+        MENUITEM "&Datum en tijd Aanpassen", ID_SHELL_CMD_ADJUST_DAT
```

Comment:
I think `Aanpassen` should be lower case on both this and the one below it.
(same for `Organizeren` etc)

Reply:
Actually it matters ^^ since it's part of typographic rules :)

To make it simple, if in Dutch people usually write titles of articles like the following:
"The Art of Writing Nice Papers"
then it's as in english, you put these capital letters.

If it's e.g. like in French, where you would write instead:
"The art of writing nice papers"
even for the title of something, then you would not put these upper case letters.